pkg/convert/json_ng.go

```diff
@@ -0,0 +1,28 @@
+package convert
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/expect-digital/translate/pkg/model"
+)
+
+func FromNgJson(b []byte) (model.Messages, error) {
+	var msg model.Messages
+
+	err := json.Unmarshal(b, &msg)
```

Comment:
`NG JSON` structure is different from the one we are using in our `model.Messages`.
You have to create map, marshal `[]byte` to that map, loop through it and manually creating `model.Message`